test/integration/api/datasets.js

```diff
@@ -64,6 +65,7 @@ describe('datasets and entities', () => {
           .set('X-Extended-Metadata', 'true')
           .expect(200)
           .then(({ body }) => {
+            body[0].should.be.an.ExtendedDataset();
             body.map(({ createdAt, lastEntity, ...d }) => {
               createdAt.should.not.be.null();
               lastEntity.should.not.be.null();
```

Comment:
We don't really need these lines (the lines in the block following this line) if we have the ExtendedDataset assertion